internal/cmd/asyncapi/command_import_test.go

```diff
@@ -0,0 +1,108 @@
+package asyncapi
+
+import (
+	v3 "github.com/confluentinc/ccloud-sdk-go-v2/kafkarest/v3"
+	"github.com/stretchr/testify/require"
+	spec2 "github.com/swaggest/go-asyncapi/spec-2.4.0"
+	"testing"
+)
+
+func TestAddTopic(t *testing.T) {
+	_, err := newCmd()
+	require.NoError(t, err)
+	kb := KafkaBinding{
+		XPartitions: 1,
+		XReplicas:   3,
+		XConfigs: BindingsXConfigs{
+			CleanupPolicy:     "delete",
+			DeleteRetentionMs: 8.64e+07,
+			//ConfluentValueSchemaValidation: "true",
+		},
+	}
+	//If topic does not exist
+	err = addTopic(details, "testTopic", kb, false)
+	require.NoError(t, err)
+	//If topic exists & overwrite is false
+	details.topics = append(details.topics, v3.TopicData{TopicName: "testTopic"})
+	err = addTopic(details, "testTopic", kb, false)
+	require.NoError(t, err)
+}
+
+func TestResolveSchemaType(t *testing.T) {
+	require.Equal(t, resolveSchemaType("avro"), "AVRO")
+	require.Equal(t, resolveSchemaType("json"), "JSON")
+	require.Equal(t, resolveSchemaType("proto"), "PROTOBUF")
+}
+
+func TestRegisterSchema(t *testing.T) {
+	_, err := newCmd()
+	require.NoError(t, err)
+	components := Components{
+		Messages: map[string]Message{
+			"TestTopicMessage": Message{
+				ContentType:  "application/json",
+				SchemaFormat: "application/schema+json;version=draft-07",
+				Payload:      `{"type": "string"}`,
+			},
+		},
+	}
+	id, err := registerSchema(details, "testTopic", components)
+	require.NoError(t, err)
+	require.Equal(t, int32(100001), id)
+}
+
+func TestUpdateCompatibility(t *testing.T) {
```

Comment:
The test should have the same name as the function
```suggestion
func TestUpdateSubjectCompatibility(t *testing.T) {
```